src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const cost = 40 * days;
+  const threeDays = 3;
+  const sevenDays = 7;
```

Comment:
```suggestion
  const threeDays = 3;
  const sevenDays = 7;
```
Якщо кількість днів зміниться, то назви threeDays та sevenDays вже не будуть відповідати дійсності. Давай придумаємо більш загальну і логічну назву для них